src/lib/csrf.ts

```
@@ -0,0 +1,124 @@
+  const array = new Uint8Array(length);
+  
+  // Try Web Crypto API first (available in modern browsers and Node.js 16+)
+  if (typeof globalThis !== 'undefined' && globalThis.crypto && globalThis.crypto.getRandomValues) {
+    globalThis.crypto.getRandomValues(array);
+    return Array.from(array, byte => byte.toString(16).padStart(2, '0')).join('');
+  }
+  
+  // Try Node.js crypto module (dynamic import for server-side compatibility)
+  try {
+    // Use dynamic import to avoid bundler issues
+    const crypto = eval('require')('crypto');
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)